ibis/tests/test_api.py

```diff
@@ -26,7 +26,7 @@ def test_backends_tab_completion():
 
 
 def test_missing_backend():
-    msg = "If you are trying to access the 'foo' backend"
+    msg = "module 'ibis' has no attribute 'foo'."
```

Comment:
Funny enough this passes because `match` accepts a regular expression and `.` of course matches any single character which includes both the literal period as well as the space that's actually there.

This is fine. We can address in a follow-up if needed.